src/main/clojure/clara/tools/inspect.clj

```diff
@@ -15,8 +15,10 @@
 ;; This is derived from the Rete-style tokens, but this token
 ;; is designed to propagate all context needed to easily inspect
 ;; the state of rules.
-(s/defrecord Explanation [matches :- [[(s/one s/Any "fact") (s/one schema/Condition "condition")]] ; Fact, condition tuples
-                           bindings :- {s/Keyword s/Any}]) ; Bound variables
+(s/defrecord Explanation [matches :- [{:result s/Any
```

Comment:
Is ```:result``` the best field name here? This could be an arbitrary fact from a non-accumulator node (rather than an accumulator result)...so perhaps we just call it ```:fact```? 

Either way, I think a more complete definition of what each of these fields could contained would be helpful to users.